packages/rocketchat-slashcommand-tableflip/tableflip.js

```diff
@@ -0,0 +1,20 @@
+/*
+* Tableflip is a named function that will replace /Tableflip commands
+* @param {Object} message - The message object
+*/
+
+
+function Tableflip(command, params, item) {
+	if (command === 'Tableflip') {
+		var msg;
+
+		msg = item;
+		msg.msg = params + ' ¯(╯°□°）╯︵ ┻━┻';
+		Meteor.call('sendMessage', msg);
+	}
+}
+
+RocketChat.slashCommands.add('Tableflip', Shrug, {
```

Comment:
Also lowercase here, for consistancy.  As well as use TableFlip instead of Shrug here.  Otherwise typing /Tableflip will result in it showing the shrug.